src/DDTrace/Integrations/Laravel/V4/LaravelProvider.php

```diff
@@ -45,30 +42,21 @@ public function register()
             return;
         }
 
-        $appName = $this->getAppName();
+        $appName = self::getAppName();
+        Bootstrap::tracerAndIntegrations();
```

Comment:
Starting a conversation here as from a functional standpoint this is perfectly legit :)

I am not sure that we should invoke `Bootstrap::tracerAndIntegrations();` here. I believe we should assume that if we are here the tracer has already been bootstrapped, being it automatically by the auto-instrumentation or manually by the user calling `Bootstrap::tracerAndIntegrations();` at the very begin of the script.

Invoking this script has a cost as integrations not previously loaded are tried again.

WDYT? (Note here and in the providers/bundles below)

Reply:
My point is that we should do nothing here :)
Did you add it because somehow we got to that point and no root span was created or this is justa  safety net?

Reply:
Ah gotcha! Yeah, I was thinking safety net, but maybe we don't need it since maybe we drop support for manual instrumentation in the future? :)